hyppo/tools/tests/test_common.py

```diff
@@ -185,7 +185,7 @@ def test_diskern(self):
                 l1, (n - 1, n + 1), (l1.itemsize * (n + 1), l1.itemsize)
```

Comment:
why are you modifying this file?

Reply:
@sampan501 I ran black and it automatically made these and all following changes